xds/internal/xdsclient/xdsresource/unmarshal_cds.go

```diff
@@ -74,9 +75,21 @@ func unmarshalClusterResource(r *anypb.Any, f UpdateValidatorFunc, logger *grpcl
 }
 
 const (
-	defaultRingHashMinSize = 1024
-	defaultRingHashMaxSize = 8 * 1024 * 1024 // 8M
-	ringHashSizeUpperBound = 8 * 1024 * 1024 // 8M
+	defaultRingHashMinSize                = 1024
+	defaultRingHashMaxSize                = 8 * 1024 * 1024 // 8M
+	ringHashSizeUpperBound                = 8 * 1024 * 1024 // 8M
+	defaultInterval                       = 10 * time.Second
```

Comment:
`defaultInterval` is a very generic name for this const given that this file is not outlier detection specific. This file has code for unmarshaling all aspects of the `Cluster` resource.

I would prefer moving these consts into `outlierConfigFromCluster` and using literal values in the test since these are used only in the `want` section of tests and using literal values there makes the tests more readable.